connect/src/wg/cosmo/platform/v1/platform_pb.ts

```diff
@@ -774,6 +774,55 @@ export class GitInfo extends Message<GitInfo> {
   }
 }
 
+/**
+ * @generated from message wg.cosmo.platform.v1.CustomCheckContext
+ */
+export class CustomCheckContext extends Message<CustomCheckContext> {
```

Comment:
I'd give it a concrete name. What about VcsContext?

Reply:
Done